.github/workflows/validate-examples.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate AsyncAPI Examples
+
+on:
+  push:
+    branches:
+      - script-ci-spec-validation # added temporarily for testing purpose
+      # - master
+
+  pull_request_target:
```

Comment:
there are no complex things that need GH token so you can use "pull_request" event. This way will be easier to work with workflow as changes in it will affect the PR where change is done, no need to merge first

Reply:
implemented through b49f4a1